### PR TITLE
add readme and intro fix

### PR DESCRIPTION
**Pull request by pz-max**
# Closes # (if applicable).
Add figures and references to power system examples. People need to have a quick lick to Davide's zenodo repo.

## Checklist

- [x] I consent to the release of this PR's code under the AGPLv3 license and non-code contributions under CC0-1.0 and CC-BY-4.0.
- [x] I tested my contribution locally and it seems to work fine.
- [x]  Code and workflow changes are sufficiently documented.
- [x] Newly introduced dependencies are added to `envs/environment.yaml` and `doc/requirements.txt`.
- [x] Changes in configuration options are added in all of `config.default.yaml` and `config.tutorial.yaml`.
- [x] Add a test config or line additions to `test/` (note tests are changing the config.tutorial.yaml)
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

**Comment by pz-max** [comment]
Docs changes: 
image

+ Readme changes are just a figure description.

File: doc/index.rst [hunk 1]
```diff
@@ -71,9 +71,16 @@ myopic and perfect pathway expansion capabilities.
...
-.. image:: img/africa_osm_map.png
-    :width: 60%
-    :align: center
+.. list-table::
+   :width: 100%
+   :class: borderless
+
+   * - .. image:: https://camo.githubusercontent.com/9705c2fea2e816839b1f22483a3c974e035b10103c51eca686321dc082fdceaf/68747470733a2f2f666f72756d2e6f70656e6d6f642e6f72672f75706c6f6164732f6462383830342f6f726967696e616c2f31582f646466303431643162393863613866386333313066316336333933656334323661623535393463662e706e67
+     - .. image:: https://camo.githubusercontent.com/8383793b362981c3dd3a542a2eb690848b64cf84a88d32043ff90a12b5fef45e/68747470733a2f2f666f72756d2e6f70656e6d6f642e6f72672f75706c6f6164732f6462383830342f6f726967696e616c2f31582f393430623236373363666333316334613666303162373930386635343664333964363764663237652e706e67
+     - .. image:: https://camo.githubusercontent.com/4be75a66ec7771c681ba24719ce96bd7e8c3a5b1d5b9a556c65ec6c31273e254/68747470733a2f2f666f72756d2e6f70656e6d6f642e6f72672f75706c6f6164732f6462383830342f6f726967696e616c2f31582f366166303839633337366231396237326164313438653465343332366331363262393464623638662e706e67
```
Comment (davide-f) [inline doc/index.rst]: I'd avoid this for this PR, what do you think?

**Review by davide-f — COMMENTED** [review]
Thanks @pz-max !
A little comment, what do you think?

**Comment by pz-max** [comment]
Good point 👍🏽 I will use them just the links from the read e image

**Comment by pz-max** [comment]
@davide-f 
> Thanks @pz-max ! A little comment, what do you think?

Changes accepted. Added now additionally some of the "how the docs" bits in the Contribute Section. The "how the docs" page was previously fully omitted. Feel free to merge when its fine.

FYI, also changed in the readthedocs admin options that there should be an independent docs build per PR such that we can review more easily. I guess it will be build with the next PR.

**Review by davide-f — APPROVED** [review]
Great Max :) I've fixed a refuse, merging as CI passes

**Comment by pz-max** [comment]
@davide-f, I think there is a misunderstanding. I think the Readme pictures should be in the documentation. Not everybody is reading the Readme, and some non-dev people will default to the documentation. The global modelling pictures as well as the description to more model pictures is very useful (Tom B or others never find those).

So, my suggestion with the openmod image links was intended and, in my opinion, that is not a big overhead. This PR has nothing to do with the general docs picture architecture suggestion in Discord. If you agree, can you revert your commit? Happy to discuss in the next dev meeting

**Comment by davide-f** [comment]
Agree @pz-max , apologies; I reset the previous change.
As CI passes, it gets merged :)

**Pull request merged**